planning/README.md

```diff
@@ -33,7 +33,7 @@
 |Firewyni    |   18:00-23:00   |   09:00-14:00   |    09:00-12:00  |   15:00-18:00   |  10:00-15:00 |
 |Hajir       |   09:00-13:00   |   09:00-13:00   |    09:00-13:00  |   09:00-13:00   |  09:00-13:00 |
 |Inggrit     |                 |                 |                 |                 |              |
-|Laura       |                 |                 |                 |                 |              |
+|Laura       |  22:00-24:00    |  22:00-24:00    |   22:00-24:00   |   22:00-24:00   |  22:00-24:00 |
```

Comment:
Rest of hours during weekend.

Reply:
Then your total of hours in the week is 10 hours.